docs/DeveloperGuide.md

```diff
@@ -51,8 +51,8 @@ The bulk of the app's work is done by the following four components:
 * [**`Logic`**](#logic-component): The command executor.
 * [**`Model`**](#model-component): Holds the data of the App in memory.
 * [**`Storage`**](#storage-component): Reads data from, and writes data to, the hard disk.
-* [**`Database`**](#database-component) : Parses data from within the App.
 
+[**`Database`**](#database-component) parses data from within the App on startup. This data is used to support user input validation according to the business logic.
 [**`Commons`**](#common-classes) represents a collection of classes used by multiple other components.
 
```

Comment:
![image](https://github.com/AY2324S1-CS2103T-T13-0/tp/assets/97437396/bc27c040-ede2-4254-b4e8-ec7a8eb54a23)
Not sure if github's view is accurate, but commons should be on a newline.

Reply:
I put Database on the newline also because it is not in the diagram below